src/modules/interaction.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Kindly note that linters are to ensure that you write clean and readable code. So it will be better if you don't disable any linter configuration. So kindly fix the linters instead of disabling it 👍🏽.